node/Cargo.toml

```diff
@@ -43,6 +43,7 @@ diesel = "1.4.8"
 fail = "0.5"
 http = "0.2.5" # must be compatible with the version rust-web3 uses
 prometheus = { version ="0.13.0", features = ["push"] }
+json-structural-diff = {version = "0.1", features = ["colorize"] }
```

Comment:
This is a new dependency for diffing JSONs in a structured/canonical way. 
Although named `"colorize"`, this feature is only related to pretty-printing the diff output to the user